config/stage.js

```diff
@@ -28,4 +28,6 @@ module.exports = {
       ],
     },
   },
+
+  allowErrorSimulation: true,
```

Comment:
Ah, okay, my comment about just checking `isDevelopment` isn't quite fair here, because it's off on stage but we want error simulation.